pkg/test/harness.go

```diff
@@ -218,23 +199,40 @@ func (h *Harness) Run() {
 
 	defer h.Stop()
 
-	if h.StartKUDO || h.StartControlPlane {
-		if err := h.RunKUDO("../../config/crds/"); err != nil {
-			h.T.Fatal(err)
-		}
+	cl, err := h.Client(false)
+	if err != nil {
+		h.T.Fatal(err)
 	}
 
-	cl, err := h.Client()
+	dClient, err := h.DiscoveryClient()
 	if err != nil {
 		h.T.Fatal(err)
 	}
 
-	dClient, err := h.DiscoveryClient()
+	// Install CRDs
+	crds, err := testutils.InstallManifests(context.TODO(), cl, dClient, h.CRDDir)
 	if err != nil {
 		h.T.Fatal(err)
 	}
 
-	if err := testutils.InstallManifests(context.TODO(), cl, dClient, h.ManifestsDir); err != nil {
+	if err := testutils.WaitForCRDs(dClient, crds); err != nil {
```

Comment:
why is it once manifests and once crds? Can both methods be manifests or CRDs?

Reply:
No, we would fail attempting to create the Frameworks and FrameworkVersions since they don’t exist yet.

Reply:
I meant, why is this method not named WaitForManifests? 🤔

Reply:
It is specifically looking for the newly created types to be available for use in the API. Not resource readiness.